ksp_plugin/interface.cpp

```diff
@@ -800,30 +817,72 @@ void principia__InsertOrKeepLoadedPart(
     PartId const part_id,
     char const* const name,
     double const mass_in_tonnes,
+    XYZ const moments_of_inertia_in_tonnes,
+    WXYZ const principal_axes_rotation,
     char const* const vessel_guid,
     int const main_body_index,
     QP const main_body_world_degrees_of_freedom,
     QP const part_world_degrees_of_freedom,
+    WXYZ const part_rotation,
     double const delta_t) {
   journal::Method<journal::InsertOrKeepLoadedPart> m(
       {plugin,
        part_id,
        name,
        mass_in_tonnes,
+       moments_of_inertia_in_tonnes,
+       principal_axes_rotation,
        vessel_guid,
        main_body_index,
        main_body_world_degrees_of_freedom,
        part_world_degrees_of_freedom,
+       part_rotation,
        delta_t});
   CHECK_NOTNULL(plugin);
+
+  // We build the inertia tensor in the principal axes and then transform it to
+  // World.
+  using PartPrincipalAxes = Frame<serialization::Frame::PhysicsTag,
+                                  serialization::Frame::PRINCIPAL_AXES, false>;
+
+  auto const moments_of_inertia =
+      FromXYZ<R3Element<MomentOfInertia>>(moments_of_inertia_in_tonnes);
```

Comment:
I'm not really comfortable with a non-SI-unit FromXYZ; use `R3Element<double>` and multiply by the appropriate unit.

Reply:
Done, in a slightly different manner.